### PR TITLE
Add translation for indexPage and layout withBanner

### LINKED ISSUE #173: [Layout] Fix/Imrpove withBanner

### PR DESCRIPTION
**Pull request by leboiss**
Close #173

### CODE REVIEW

**Review by Alketta — COMMENTED** [review]
Thank you for the translation !

File: packages/i18n/en.json [hunk 1]
```diff
@@ -38,10 +38,33 @@
...
+  },
   "pages": {
     "signIn": {
       "title": "Welcome"
     },
+    "indexPage": {
+      "presentation": "LocoKit is an open source LCAP (Low Code Application Platform) that provides Database Management as a Spreadsheet with an App Builder",
+      "buildData": "Build your data",
+      "shareSecurely": "and share it securely",
+      "createDatabase": "Create your database, according to your job. Share your data, according to the permissions you set !",
```
[the remark this answers is not in the document]
Comment (leboiss) [reply packages/i18n/en.json]: done

File: packages/i18n/en.json [hunk 2]
```diff
@@ -38,10 +38,33 @@
       "cancel": "Cancel"
     }
   },
+  "layouts": {
+    "withBanner": {
+      "openMenu": "Ouvrir le menu",
```
[the remark this answers is not in the document]
Comment (leboiss) [reply packages/i18n/en.json]: done

File: packages/nuxt-locokit/src/pages/index.vue [hunk 3]
```diff
@@ -40,15 +39,17 @@
         <h1
           class="text-4xl font-bold tracking-tight text-primary sm:text-5xl md:text-6xl"
         >
-          <span class="block">Share your data</span>
-          <span class="block text-secondary"> by crafting your app</span>
+          <span class="block">
+            {{ $t('pages.indexPage.shareData') }}
```
[the remark this answers is not in the document]
Comment (leboiss) [reply packages/nuxt-locokit/src/pages/index.vue:43]: text is still in wip, i will change it later

**Review by Alketta — APPROVED** [review]

**Pull request merged**